db/migrate/20221020001400_remove_preservation_policies.rb

```diff
@@ -0,0 +1,9 @@
+class RemovePreservationPolicies < ActiveRecord::Migration[6.1]
+  def change
+    remove_foreign_key :preserved_objects, :preservation_policies
+    drop_table :preservation_policies_zip_endpoints
+    drop_table :moab_storage_roots_preservation_policies
+    drop_table :preservation_policies
+    remove_column :preserved_objects, :preservation_policy_id
```

Comment:
@jmartin-sul Here is the key bit to make sure we've got right by testing in QA and/or stage environments. As part of that work, I've run `pg_dump` on the DB in QA as a "before" snapshot, and if you think this PR is good enough, I could deploy it to QA and then run an "after" snapshot, and then I can diff the snapshots and write up some analysis.

Reply:
@jmartin-sul Sounds good. What would you think about having this falling under the FR umbrella?

Related, the two DB snapshots are now available on preservation-catalog-qa-01 in ~:

```
-rw-rw-r--   1 pres pres 17766963 Nov 15 16:17 pres_2022-11-15T16:17:45-0800.sql
-rw-rw-r--   1 pres pres 17679813 Nov 15 16:20 pres_2022-11-15T16:20:04-0800.sql
```

The latter is ~285 lines smaller than the former, and AFAICT (it's hard to make sense of parts of the diff) it drops the policies w/o dropping anything we care about, but it'd be good for someone else to verify this. This PR is now 💚 and was deployed to QA. Would be great if we didn't make to rebase it again!

Reply:
> What would you think about having this falling under the FR umbrella?

seems reasonable to me